web_form_banner/tests/test_web_form_banner.py

```diff
@@ -0,0 +1,213 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+
+from odoo.tests.common import TransactionCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestFieldsViewGetPartnerBanner(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.Partner = cls.env["res.partner"]
+        cls.Rule = cls.env["web.form.banner.rule"]
+        existing_rules = cls.Rule.search([])
+        cls._disabled_rule_ids = existing_rules.ids
+        if existing_rules:
+            existing_rules.write({"active": False})
+        IrModel = cls.env["ir.model"]
+        Rule = cls.env["web.form.banner.rule"]
+        partner_model = IrModel._get("res.partner")
+        model_id = partner_model.id
```

Comment:
I'd prefer this. No need to define `Rule` as it doesn't save lines.
```suggestion
        partner_model_id = cls.env["ir.model"]._get("res.partner").id
```